apps/web-giddh/src/app/shared/amount-field/amount-field.component.ts

```diff
@@ -0,0 +1,37 @@
+import { Component, Input, OnChanges, OnInit, SimpleChanges } from '@angular/core';
+import { GeneralService } from '../../services/general.service';
+
+@Component({
+    selector: 'amount-field',
+    templateUrl: './amount-field.component.html',
+    styleUrls: ['./amount-field.component.scss']
+})
+
+export class AmountFieldComponent implements OnInit, OnChanges {
+    @Input() public amount: any;
```

Comment:
Add JS doc in all the members and methods

Reply:
Done